apps/charterafrica/src/components/Explainers/Explainer.js

```diff
@@ -0,0 +1,70 @@
+import RichTypography from "@/commons-ui/core/RichTypography";
+import { Figure } from "@commons-ui/next";
+import { Grid, Divider } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import RichText from "../RichText";
+
+function Explainer({ image, description, title }) {
+  return (
+    <>
+      <Grid display="flex" justifyContent="center" sx={{ p: 8 }} container>
```

Comment:
```suggestion
      <Grid container justifyContent="center">
```
Grid already uses display flex.